test/unitTests.js

```diff
@@ -166,3 +166,40 @@ describe('Profile Test for Correct email, age, gender, instagram username, and p
 
 // Unit Tests for File Upload Feature
 
+
+//Unit Tests for Map Markers
+
+const city3 = "Toronto"
+const city4 = "Rome"
+const groupSize3 = 12
+const groupSize4 = 50
+const startDate3 = new Date("2020-11-30")
+const endDate3 = new Date("2020-12-02")
+
+describe('Test number of attractions returned: Toronto with group size 12',function(){
+    it('Recommended Attractions should return 4', async function(){
+        const response = await RecommendController.recommend(city3,groupSize3,startDate3,endDate3)
+        assert.equal(response.length,4);
+    })
+});
+
+describe('Test number of attractions returned: Toronto with group size 50',function(){
+    it('Recommended Attractions should return 4', async function(){
+        const response = await RecommendController.recommend(city3,groupSize4,startDate3,endDate3)
+        assert.equal(response.length,4);
+    })
+});
```

Comment:
These tests look great. I understand that the response length is returning 4 as there are 4 parameters being passed into the method being tested - is this the correct understanding? I like how you incorporated two different group sizes to ensure you are testing more than one case. Well done!